modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -298,7 +298,16 @@ ifdef::ibm-z,ibm-z-kvm[]
 <15> Add the `additionalTrustBundle` parameter and value. The value must be the contents of the certificate file that you used for your mirror registry. The certificate file can be an existing, trusted certificate authority or the self-signed certificate that you generated for the mirror registry.
 endif::ibm-z,ibm-z-kvm[]
 ifndef::openshift-origin[]
-<16> Provide the `imageContentSources` section from the output of the command to mirror the repository.
+<16> Provide the `imageContentSources` section according to the output of the command you used to mirror the repository. 
++
+[IMPORTANT]
+====
+Provide the input according to the command you used for mirroring:
```

Comment:
Maybe this IMPORTANT admonition could be revised a little bit so reduce the repetitiveness of the first sentence and <16>th entry above:

```suggestion
[IMPORTANT]
====
* If you are using the `oc adm release mirror` command, refer to the `imageContentSources` section found in its output.
* <insert_similar_text_here>
====
```

or maybe

```suggestion
[IMPORTANT]
====
* When using the `oc adm release mirror` command, refer to the `imageContentSources` section found in its output.
* <insert_similar_text_here>
====
```

Reply:
@stevsmit Thank you, this is done!